openomni/bin/decode_omni

```diff
@@ -11,22 +11,33 @@ from openomni.packet import Packet
 
 def main(options=None):
     parser = argparse.ArgumentParser(description="Print out structured version of packet (given as a hex string).")
-    parser.add_argument("data", metavar="data", type=str, nargs="+",
+    parser.add_argument("--infile", type=argparse.FileType("r"),
+            help="Input text-formatted file.")
+    parser.add_argument("data", metavar="data", type=str, nargs="?",
+    parser.add_argument("data", metavar="data", type=str, nargs="*",
```

Comment:
Is this line (nearly) duplicated on purpose? I'm not familiar enough with this api to know if this is ok.